feat(dedibox-domains): removed domain name ownership change MTA-7391 (#6809)
scaleway/docs-content: pages/dedibox-domains/how-to/manage-dns-zones.mdx

---

> title: How to manage a domain DNS zones

Reviewer: ```suggestion
title: How to manage the DNS zones of a domain
```

> description: This page explains how to manage a domain DNS zones at Scaleway Dedibox

Reviewer: ```suggestion
description: This page explains how to manage the DNS zones of a domain on Scaleway Dedibox
```

| **A**       | Allows you to link a domain name with an IP address (IPv4) (`www.online.net.` IN A 212.27.35.195). |
| **AAAA**    | Allows you to link a domain name with an IP address (IPv6) (irc.online.net. IN AAAA 2a01:e0b:1:f:2d0:b7ff:fea9:4da8). |
| **CNAME**   | Allows the mapping between a DNS name and another DNS name (sql.domain.ext. IN CNAME mysql1.online.net.). |

> | **MX**      | Allows you to set the responsible mail server for a domain (online.net. IN mx.online.net.). The DNS name MUST be an IPv4 and can not be an IP address or a CNAME. Priority defines the time in which the servers are contacted if there are more than one. |

Reviewer 2: ```suggestion
| **MX**      | Allows you set the mail server responsible for a domain (e.g., online.net. IN MX mail.online.net.). The DNS name must be an IPv4, not an IP address or a CNAME. If multiple mail servers are configured, the priority value determines the order in which they are contacted. |
```

> | **MX**      | Allows you to set the responsible mail server for a domain (online.net. IN mx.online.net.). The DNS name MUST be an IPv4 and can not be an IP address or a CNAME. Priority defines the time in which the servers are contacted if there are more than one. |

Reviewer: ```suggestion
| **MX**      | Allows you to set the responsible mail server for a domain (online.net. IN mx.online.net.). The DNS name MUST be an IPv4 and cannot be an IP address or a CNAME. Priority defines the order in which the servers are contacted if there are more than one. |
```

| **TXT**     | This record can store a string of 1024 characters. |
| **SRV**     | This registration enables automatic discovery services. This type of record is reserved for expert users only. |

<Message type="important">

>   - Experts only should modify DNS zone.

Reviewer: ```suggestion
  - Experts only should modify a DNS zone.
```

  - Errors in the configuration of a DNS zone may lead to the unavailability of your website or emails.

>   - Scaleway does not provide support for web hostings or domain names with DNS zones modified by the user.

Reviewer: ```suggestion
  - Scaleway does not provide support for web hostings or domain names when the DNS zones have been modified by the user.
```

## How to use BIND configuration mode (Advanced)

> To facilitate the management of your zones, reserved for experts, Scaleway provides you with the option of sending and downloading your DNS zone in the BIND format. This is the format mostly used for the configuration of a BIND DNS Server, and allows you to re-import your zones if they were previously in this format, or if you are used to it.

Reviewer: ```suggestion
To facilitate the management of your zones, reserved for experts, Scaleway provides you with the option of sending and downloading your DNS zone in the BIND format. This is the format mostly used for the configuration of a BIND DNS Server. Use BIND to re-import your zones if they were previously in this format, or if you want to work in a format that you are already familiar with.
```
